service/controller/clusterapi/v19/resources/app/current.go

```diff
@@ -0,0 +1,53 @@
+package app
```

Comment:
Resource is aligned with the chartconfig resource in clusterapi but uses the generic app resource.